entry

```diff
@@ -110,7 +110,8 @@ helm_content_decode() {
 	set +e
 }
 
-HELM="helm_v3"
+export HELM_TLS_CA_CERT=/var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+HELM="helm_v3 --ca-file=/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"
```

Comment:
You can't do this; there are multiple tests in this file that do `if [[ "${HELM}" == "helm_v3" ]]; then` - does setting the environment variable not work reliably?

Reply:
I did this originally because the ENV parameter was not in the helm v3 docs and I thought it only applied to helm v2 while the --ca-file flag is in the v3 docs. Tested working with just the ENV and it works so 👍 thank you